mods/ra/rules/misc.yaml

```diff
@@ -208,6 +210,8 @@ MINE:
 	RadarColorFromTerrain:
 		Terrain: Ore
 	SeedsResource:
+	EditorTilesetFilter:
+		Categories: Mine
```

Comment:
This will put Ore and Gem mines in the same category as explosive mines. I suggest Resource spawns for the ore/gem/rail mines here.